authentication/views/auth.py

```diff
@@ -14,9 +14,22 @@
 from authentication.models.activation import Activation
 from authentication.views import auth
 
+from authentication.models.users import Teacher
+
 
 def index(request):
```

Comment:
This is the root of the entire app - what I've tried to start here is the ability to route a user to the correct application based on the type of user that is logged in. In this case I've only set it up for Teachers. If its a teacher redirect them to the school app. But we can continue to add more control for each kind of User

Reply:
this is great. I was thinking about this.